README.md

```diff
@@ -8,99 +8,166 @@ _user_ not a _developer_.
+evaluated by the shell with the help of the operating system and results will
+be printed for you. To trigger evaluation of your command you strike the
+`return` or `enter` keys.
 
-Typically you'll be given some text output and a command prompt. The prompt
-will receive commands. You tell the shell you're done typing and that it should
-consider your request by hitting `return` or `enter`. After receiving your
-command the shell will respond with the information you requested, or will
-signal that it did your bidding (say renaming a file), or it will print an
-error message. Then you'll be given another prompt and can do it again, and
-again, and again.... This cycle can be called a "prompt-evaluation-response
-cycle."
+### See Shells in the Wild
 
-Here's the Learn In-Browser IDE's shell. Since the shell environment is so
-important for us, whenever you open the IDE the command prompt is also shown.
-You will interacting with the shell when you run commands like `learn` and
-`learn submit`.
+Since the terminal environment is so important for us, whenever you open the
+IDE, the LearnIDE terminal is automatically shown.  You will interact with the
+shell through this terminal when you run commands like `learn` and `learn
+submit`.
 
 <img alt="Learn In-Browser IDE Shell" src="https://curriculum-content.s3.amazonaws.com/web-development/intro-the-shell/learn_shell.png" width=595>
 
-Here's the Terminal application in MacOSX running a shell. You can launch it by
-navigating with the Finder to `Applications` &rarr; `Utilities` &rarr;
-`Terminal`.
+Here's the Terminal application in MacOSX running a terminal program. You can
+launch it by navigating with the Finder to `Applications` &rarr; `Utilities`
+&rarr; `Terminal`. There are other third-party terminal programs too. They all
+provide a way of using a shell. Liking one over the other is a matter of
+preference.
 
 <img alt="MacOSX Shell" src="https://curriculum-content.s3.amazonaws.com/web-development/intro-the-shell/osx_shell.png" width=595>
 
-Here we see the prompt-evaluation-response cycle, here we
-are providing the command `echo` which, well, repeats the command back to the
-screen (the "response").
+Here we see the prompt-evaluation-response cycle, here we are providing the
+command `echo` which, well, repeats the command back to the terminal (the
+"response").
 
 <img alt="MacOSX Shell response" src="https://curriculum-content.s3.amazonaws.com/web-development/intro-the-shell/prompt_response.png" width=595>
 
+### Erroneous Use
+
 You may sometimes hear the shell referred to as your "command line" or
 "terminal" or "console". There are some subtle differences between each of
 those terms, but by and large you can think of these as referring to ways of
-interacting with the shell.
+interacting with the shell (as presented by the terminal).  While some uses are
+_technically_ erroneous, it's a common utterance and you should not ruin a
+party by correcting everyone's imprecise naming.
```

Comment:
This is hilarious. Perhaps add a line in about saying at Learn, we'll use "terminal" to describe this way of interacting with your computer?